lib/galaxy/datatypes/binary.py

```diff
@@ -1107,6 +1107,63 @@ def display_peek(self, dataset):
             return "Cool (HDF5) file (%s)." % (nice_size(dataset.get_size()))
 
 
+class MCool(H5):
```

Comment:
Should this derive from Cool?

Reply:
That is unclear to me. The structure of the mcool file is a series of cool files nested as groups within the mcool file, meaning that the only function specific to the cool class (the sniffer) doesn't apply to the mcool class. But for the sake of suggesting the relationship, I guess that makes sense.

Reply:
Turns out this should not derive from Cool as this causes a failure of the sniffer